runtime/metrics_api.go

```diff
@@ -111,6 +112,9 @@ func (m *MetricsApi) background() {
 			m.freq = f
 			resetTicker(f)
 
+		case <-context.Background().Done():
```

Comment:
I don't think this will ever fire -- nil channel.

https://golang.org/src/context/context.go